src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
 // write code here
+const inputs = document.querySelectorAll('input');
+
+inputs.forEach(el => {
+  const label = document.createElement('label');
+  const inputName = el.getAttribute('name').split(/(?=[A-Z])/);
```

Comment:
you can write just input.name, no need to use getAttribute

```suggestion
inputs.forEach(input => {
  const label = document.createElement('label');
  const inputName = input.name.split(/(?=[A-Z])/);
```